src/selectstar.py

```diff
@@ -98,8 +100,15 @@ def __get_warehouse_links(self, dbt_models: list[DbtModel]):
 
             for link in found_links["results"]:
                 warehouse_link = WarehouseLink(link)
-                warehouse_link.set_table(self.__get_table(warehouse_link.guid))
-                model.warehouse_links.append(warehouse_link)
+                found_table = self.__get_table(warehouse_link.guid)
+                if found_table:
+                    warehouse_link.set_table(found_table)
+                    model.warehouse_links.append(warehouse_link)
+                else:
+                    log.warning(
+                        f"   A warehouse link was found, but the target DWH table was not found."
```

Comment:
why do you need 4 spaces at the beginning of the warning text?

Reply:
I bit of indentation to help the customer to read the log:

```
2024-01-18 12:41:39,069 INFO Starting Dbt Impact Report by Select Star.
2024-01-18 12:41:39,069 INFO These are the current values of the required settings
2024-01-18 12:41:39,069 INFO    SELECTSTAR_API_URL = 'https://api.production.selectstar.com'
2024-01-18 12:41:39,070 INFO    SELECTSTAR_WEB_URL = 'https://app.selectstar.com'
2024-01-18 12:41:39,070 INFO    SELECTSTAR_API_TOKEN = '**HIDDEN**'
2024-01-18 12:41:39,070 INFO    SELECTSTAR_DATASOURCE_GUID = 'ds_GeypMekj2qMizMRT4gMqBs'
2024-01-18 12:41:39,070 INFO    GIT_PROVIDER = 'github'
2024-01-18 12:41:39,070 INFO    GIT_CI = 'False'
2024-01-18 12:41:39,070 INFO    GIT_REPOSITORY = 'selectstar/dbt-selectstar'
2024-01-18 12:41:39,070 INFO    GIT_REPOSITORY_TOKEN = '**HIDDEN**'
2024-01-18 12:41:39,070 INFO    PULL_REQUEST_ID = '362'
2024-01-18 12:41:39,070 INFO Is this a CI execution? False
2024-01-18 12:41:39,070 INFO Getting the list of changed models using GIT API.
2024-01-18 12:41:39,725 INFO Found models: [('erp_transaction.sql', 'added'), ('table_model.sql', 'modified'), ('view_model.sql', 'modified')]
2024-01-18 12:41:39,726 INFO Getting the lineage for each dbt model.
2024-01-18 12:41:39,726 INFO  Fetching the dbt models GUID
2024-01-18 12:41:39,726 INFO   Fetching GUID for the models: 'erp_transaction.sql,table_model.sql,view_model.sql' self.datasource_guid='ds_GeypMekj2qMizMRT4gMqBs'
2024-01-18 12:41:40,522 INFO  Fetching the dbt models warehouse links
2024-01-18 12:41:40,522 INFO   Fetching warehouse links for model.guid='ta_4BGPSvmmPfigVo48LU7JEf' model.filename='table_model.sql'
2024-01-18 12:41:42,120 INFO   Fetching warehouse links for model.guid='ta_7hi6J7nTf5DrQKA8KqZVKv' model.filename='view_model.sql'
2024-01-18 12:41:43,903 WARNING    A warehouse link was found, but the target DWH table was not found. Was it removed/deactivated? Missing table guid = ta_N3RBBaiewWnDBRy7taaFdP
2024-01-18 12:41:43,903 INFO  Fetching the dbt models full lineage
2024-01-18 12:41:43,903 INFO   Fetching lineage for element.guid='ta_4BGPSvmmPfigVo48LU7JEf'
2024-01-18 12:41:44,201 INFO   Fetching lineage for element.guid='ta_eH82n7q4eW6KgSiD96udU2'
2024-01-18 12:41:44,484 INFO   Fetching lineage for element.guid='ta_7hi6J7nTf5DrQKA8KqZVKv'
2024-01-18 12:41:44,711 INFO Creating the report.
2024-01-18 12:41:44,712 INFO Searching for previous impact report.
2024-01-18 12:41:45,030 INFO Previous impact report found. id=1896288024 url=https://github.com/selectstar/dbt-selectstar/pull/362#issuecomment-1896288024.
2024-01-18 12:41:46,565 INFO Previous impact report updated. id=1896288024 url=https://github.com/selectstar/dbt-selectstar/pull/362#issuecomment-1896288024.
2024-01-18 12:41:46,565 INFO Dbt Impact Report has ended, bye!
```